src/hiero_sdk_python/client/client.py

```
@@ -240,6 +245,37 @@ def get_tls_root_certificates(self) -> Optional[bytes]:
         Retrieve the configured root certificates for TLS connections.
         """
         return self.network.get_tls_root_certificates()
+    
+    def set_default_max_query_payment(self, max_query_payment: Union[int, float, Decimal, Hbar]) -> "Client":
```

Ruff (0.14.10)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)